core/templates/dev/head/filters.js

```diff
@@ -336,4 +336,41 @@ oppia.filter('summarizeDefaultOutcome', ['$filter', function($filter) {
     }
     return summary;
   }
-}])
+}]);
+
+// Filter that summarizes large number to nearly four significant figures and
```

Comment:
Filter that summarizes a large number to a decimal followed by
the appropriate metric prefix (K, M or B). For example, 167656
becomes 167.7K.